stack/stack_mac.yaml

```diff
@@ -277,7 +277,7 @@ pybind11:
 
 ecbuild:
   build: YES
-  version: 3.6.1
+  version: 2021.08.0
```

Comment:
Do you know what the difference is between the year version and the v3.x.x? I see they have a bunch of tags that kind of alternate between.

Reply:
I could not yet find any information about the differences. The tag 2021.08.0 seems to correspond to 3.6.3 (maybe identical?); a VERSION file in 2021.08.0 has 3.6.3, and also these tags seem to be similar in github, at least from the first glance.

Reply:
Tested the software build for fms 3.6.3 -successful. This could be reverted to 3.x.x format.